src/renderer/pages/Operations/components/Details.tsx

```diff
@@ -47,97 +53,68 @@ const Details = ({ tx, account, connection, isCardDetails = true }: Props) => {
 
   return (
-    <dl className="flex flex-col gap-y-1 w-full">
-      {isCardDetails && description && (
-        <div className={DescriptionBlockStyle}>
-          <FootnoteText as="dt" className="text-text-tertiary">
-            {t('operation.details.description')}
-          </FootnoteText>
-          <FootnoteText as="dd" className={cnTw('break-words', valueClass)}>
-            {description}
-          </FootnoteText>
-        </div>
-      )}
+    <dl className="flex flex-col gap-y-4 w-full">
       {cancelDescription && (
         <div className={DescriptionBlockStyle}>
           <FootnoteText as="dt" className="text-text-tertiary">
             {t('operation.details.rejectReason')}
           </FootnoteText>
-          <FootnoteText as="dd" className={cnTw('break-words', valueClass)}>
+          <FootnoteText as="dd" className="break-words">
             {cancelDescription}
           </FootnoteText>
         </div>
       )}
 
       {account && wallet && (
-        <DetailRow label={t('operation.details.multisigWallet')} className={valueClass}>
-          <WalletCardSm
-            wallet={wallet}
-            accountId={account.accountId}
-            addressPrefix={addressPrefix}
-            explorers={explorers}
-            className="-mr-2 max-w-none"
-          />
+        <DetailRow label={t('operation.details.multisigWallet')}>
+          <div className="flex gap-x-2 items-center max-w-none">
+            <WalletIcon type={wallet.type} size={16} />
+            <FootnoteText>{wallet.name}</FootnoteText>
+          </div>
         </DetailRow>
       )}
 
-      {isXcmTransaction(tx.transaction) && (
-        <>
-          {isCardDetails && (
-            <DetailRow label={t('operation.details.fromNetwork')} className={valueClass}>
-              <ChainTitle chainId={tx.chainId} fontClass={valueClass} />
-            </DetailRow>
-          )}
-
-          {isCardDetails && account && (
-            <DetailRow label={t('operation.details.sender')} className={valueClass}>
-              <AddressWithExplorers
-                explorers={explorers}
-                addressFont={AddressStyle}
-                type="short"
-                accountId={account.accountId}
-                addressPrefix={addressPrefix}
-                wrapperClassName="-mr-2 min-w-min"
-              />
-            </DetailRow>
-          )}
-
-          {transaction.args.destinationChain && (
-            <DetailRow label={t('operation.details.toNetwork')} className={valueClass}>
-              <ChainTitle chainId={transaction?.args.destinationChain} fontClass={valueClass} />
-            </DetailRow>
-          )}
-        </>
-      )}
-
-      {transaction?.args.dest && (
-        <DetailRow label={t('operation.details.recipient')} className={valueClass}>
+      {account && (
+        <DetailRow label={t('operation.details.' + hasSender ? 'sender' : 'account')} className="text-text-secondary">
```

Comment:
```suggestion
        <DetailRow label={t(hasSender ? 'operation.details.sender' : 'operation.details.account')} className="text-text-secondary">
```
I think this way is more secure than string concatenation